<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>613</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Futura</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,1,4">
      <item row="0" column="3" rowspan="6">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeView" name="treeView"/>
        </item>
        <item>
         <widget class="QLineEdit" name="print_lineEdit"/>
        </item>
       </layout>
      </item>
      <item row="0" column="0" rowspan="6" colspan="3">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0,0,0,0,0">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Key:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="key_spinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Data:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="data_lineedit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="insert_btn">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="find_btn">
          <property name="text">
           <string>Find</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="balance_btn">
          <property name="text">
           <string>Balance</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="preOrder">
          <property name="text">
           <string>print preorder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="inOrder_btn">
          <property name="text">
           <string>print inorder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="postOrder_btn">
          <property name="text">
           <string>print postorder</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="execTask_btn">
          <property name="text">
           <string>Execute task</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="getInfo_btn">
          <property name="text">
           <string>Get Info</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
